ote_sdk/ote_sdk/usecases/exportable_code/prediction_to_annotation_converter.py

```diff
@@ -50,7 +64,9 @@ class DetectionToAnnotationConverter(IPredictionToAnnotationConverter):
     def __init__(self, labels: List[LabelEntity]):
         self.label_map = dict(enumerate(labels))
 
-    def convert_to_annotation(self, predictions: np.ndarray) -> AnnotationSceneEntity:
+    def convert_to_annotation(
+        self, predictions: np.ndarray, metadata: Dict = None
```

Comment:
Not sure how pylint and mypy would be happy with `metadata: Dict = None`. I think it should be `metadata: Optional[Dict] = None`